.gitignore

```diff
@@ -1,4 +1,5 @@
 vendor/
 bin/
 composer.lock
-composer.phar
\ No newline at end of file
+composer.phar
+.idea/
```

Comment:
.idea should not be included in project .gitignore file. You can setup in your global .gitignore https://help.github.com/articles/ignoring-files/#create-a-global-gitignore